Week5/week5-tasks/src/CountWords.java

```diff
@@ -0,0 +1,42 @@
+import java.io.BufferedInputStream;
+import java.io.FileInputStream;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+
+public class CountWords {
+    // Create Private instance from Scanner Class
+    private static final Scanner in = new Scanner(System.in);
+    private static final String path = "/home/karim/Desktop/internship/JobHacker-System/Week5/week5-tasks/src/Lincoln.txt";
+    public static void main(String[] args) {
+        System.out.println(countWords(path));
+
+    }
+
+    public static int countWords(String fileName) {
+        int counter = 0;
+        try{
+            // Instance from file  to move pointer from first charchter
```

Comment:
remove unneeded comments. Only write comments for complex code or to describe why a code is written.